specification/dynamicstelemetry/resource-manager/Microsoft.DynamicsTelemetry/preview/2019-01-24/dynamicstelemetry.json

```diff
@@ -0,0 +1,142 @@
+{
+  "swagger": "2.0",
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "info": {
+    "title": "Dynamics Telemetry",
+    "version": "2019-01-24"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.DynamicsTelemetry/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "produces": [
```

Comment:
consumes?

Reply:
> consumes?

If you mean - does Operations action consumes some parameters - then no. 
Otherwise please clarify your question.

Kind regards.